src/server/dfly_main.cc

```diff
@@ -323,13 +324,26 @@ bool RunEngine(ProactorPool* pool, AcceptServer* acceptor) {
   auto port = GetFlag(FLAGS_port);
   auto mc_port = GetFlag(FLAGS_memcache_port);
   string unix_sock = GetFlag(FLAGS_unixsocket);
+  mode_t unix_socket_perm;
```

Comment:
Sorry, could you move `unix_socket_perm` into the `if`? It's not used outside, I missed that fact :(